src/mapeo-manager.js

```diff
@@ -89,6 +89,9 @@ export const DEFAULT_FALLBACK_MAP_FILE_PATH = require.resolve(
 export const DEFAULT_ONLINE_STYLE_URL =
   'https://demotiles.maplibre.org/style.json'
 
+// Oldest possible time, ensure it gets overwritten with any updates
+export const UNIX_EPOCH_DATE = new Date(0).toISOString()
```

Comment:
exporting this doesn't seem necessary

```suggestion
const UNIX_EPOCH_DATE = new Date(0).toISOString()
```